src/locale/locales/es/messages.po

```diff
@@ -133,7 +133,7 @@ msgstr "{0} de {1}"
 
 #: src/screens/StarterPack/StarterPackScreen.tsx:478
 msgid "{0} people have used this starter pack!"
-msgstr "¡{0} personas han usado este paquete de inicio!"
+msgstr "¡{0} personas han usado este pack de inicio!"
```

Comment:
Why should we use `pack` instead of `paquete`? we're trying to localize the app :) sounds better for me but isn't correct for most users.

Maybe we should consider another text?

Reply:
Ok! I have no strong opinion here, maybe I'm too biased as I use "pack" in Spanish way more than "paquete" 😂 I'll change it back